demo-shell/src/app.config.json

```diff
@@ -484,5 +484,10 @@
                 "exif:exif": "*"
             }
         }
+    },
+    "adf-version-manager": {
```

Comment:
The schema json need do be updated as well

Reply:
added schema update